.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+    * Question: Prefix your issue title with [QUESTION]
+-->
+
+<!--
+    This section is only for Bug and Question issues
+-->
+* Sponge Implementation: <!-- SpongeForge or SpongeVanilla -->
+* Sponge build version: <!-- Find the version using /sponge version -->
```

Comment:
Actual @dualspiral most server owners should know which Sponge flavor they are using and the build. I also believe if they do `/sponge plugins` and hover over `sponge` it returns the version number, and if they click, it provides more information.